### ISSUE TITLE
Question - Package without CDN

### ISSUE THREAD

**Issue by Armachillo** [mrackley/ModernHillbillyTabs#1]
Hello, is there a way to package this solution without using CDN?

**Comment by Sydsalimander** [comment]
Any update?

**Comment by ewidance** [comment]
No, but you ca use the one provided by office... https://docs.microsoft.com/fr-fr/sharepoint/dev/spfx/web-parts/get-started/hosting-webpart-from-office-365-cdn
